src/map/zone_entities.cpp

```diff
@@ -456,6 +456,12 @@ void CZoneEntities::DecreaseZoneCounter(CCharEntity* PChar)
     m_charList.erase(PChar->targid);
     charTargIds.erase(PChar->targid);
 
+    // Need to interupt fishing on zone out otherwise fished up mobs get stuck in hooked state
+    if (PChar->hookedFish && PChar->hookedFish->hooked == true)
```

Comment:
A problem for another day, but surely the non-nullness of a member called 'hookedFish' implies that it is indeed... Hooked?